<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="GraphOptions">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>600</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>All</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Abundance</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>neutromer spacing</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Combined</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="CalcMethodLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="GraphsLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>600</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Graphs</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CalculationMethodOptions">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Both</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Abundance</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Neutromer Spacing</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="RollUpLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Roll Up Proteins</string>
   </property>
  </widget>
  <widget class="QComboBox" name="RollUp">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yes</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="HeavyElement">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>H</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>N</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="HeavyElementLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Heavy Element</string>
   </property>
  </widget>
  <widget class="QComboBox" name="LabelPattern">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>tissue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>culture</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>global</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="LabellingPatternLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Labeling Site Pattern</string>
   </property>
  </widget>
  <widget class="QLabel" name="StepSizeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Step Size to Max Label (%)</string>
   </property>
  </widget>
  <widget class="QLabel" name="Maxlabellabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Label - Theory(%)</string>
   </property>
  </widget>
  <widget class="QLabel" name="mintimepointslabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Min Number of non-Zero Time-points</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="TheoryLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>7</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="StepSize">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="TimePoints">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveSettings">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitMenu">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ResetToDefaults">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset to Default</string>
   </property>
  </widget>
  <widget class="QLabel" name="n_isos_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string># of Neutromers to Extract</string>
   </property>
  </widget>
  <widget class="QLabel" name="lowz_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum z Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="high_z_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum z Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_window_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Window</string>
   </property>
  </widget>
  <widget class="QLabel" name="ppm_window_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ppm Window</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="lowz">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="highz">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="time_window">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
   <property name="value">
    <double>1.500000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="ppm_window">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.989999999999782</double>
   </property>
   <property name="singleStep">
    <double>5.000000000000000</double>
   </property>
   <property name="value">
    <double>20.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="n_isos">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>4</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Last_Saved_Settings">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Asymptote Calculation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fixed Asymptote Value</string>
   </property>
  </widget>
  <widget class="QComboBox" name="asymptote_calc">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>510</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Fixed</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Variable</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="FAValue">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>540</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>2.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Proliferation Adjustment</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Prolif_Adjust">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>570</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="intensity_bias_box">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="mintimepointslabel_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User Bias Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="mintimepointslabel_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Bias Calculation</string>
   </property>
  </widget>
  <widget class="QComboBox" name="bias_selection">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>User Defined</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DeuteRater Calculated</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="mintimepointslabel_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Intensity</string>
   </property>
  </widget>
  <widget class="QLabel" name="mintimepointslabel_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Spacing</string>
   </property>
  </widget>
  <widget class="QLabel" name="mintimepointslabel_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Combined</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spacing_bias_box">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>440</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="combined_bias_box">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>440</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
